framework/src/actions/SetAdaptivityOptionsAction.C

```diff
@@ -139,6 +156,14 @@ SetAdaptivityOptionsAction::act()
 
     adapt.setRecomputeMarkersFlag(getParam<bool>("recompute_markers_during_cycles"));
     if (getParam<bool>("switch_h_to_p_refinement"))
-      adapt.switchHToPRefinement();
+    {
+      auto disable_p_refinement_for_families =
+          getParam<MultiMooseEnum>("disable_p_refinement_for_families");
+      if (!isParamSetByUser("disable_p_refinement_for_families"))
+        // If the user has not set this parameter we will set a logicial default
+        disable_p_refinement_for_families =
+            "LAGRANGE NEDELEC_ONE LAGRANGE_VEC CLOUGH BERNSTEIN RATIONAL_BERNSTEIN";
```

Comment:
I agree with you. I did not want to introduce `RAVIART_THOMAS` since it does not yet exist as one of our MOOSE available families. I believe https://github.com/idaholab/moose/pull/25838 is a stepping stone for that. I will open a libMesh issue for getting that `is_hierarchic` shim

Reply:
opened https://github.com/libMesh/libmesh/issues/3705